concepticondata/conceptlists/Araujo-1996-289.tsv

```diff
@@ -146,7 +146,7 @@ Araujo-1996-289-144	144	Dance Mask			Tanzmaske	máscara de dança
 Araujo-1996-289-145	145	Dance Mast			Tanzpfahl	mastro para dança
 Araujo-1996-289-146	146	Dancing Rattle	300	RATTLE	Tanzrassel	chocalho para dança
 Araujo-1996-289-147	147	Dancing Rammer (Olyra)			Stampfrohr	tubo [de taquara] para bater (o ritmo)
-Araujo-1996-289-148	148	Shade Or Shadow	1388	SHADE	Schatten	sombra
+Araujo-1996-289-148	148	Shade Or Shadow	2275	SHADOW	Schatten	sombra
```

Comment:
This must be mapped to `1388 SHADE`, if two concept sets are identical (as marked by an entry in the ALIAS column, we use the smaller ID. So SHADE is the choice here.